thirdparty/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Directories caught by the root .gitignore's buildfile list
+!embree/common/simd/arm/
+!libpng/arm/
+!libtheora/x64/
```

Comment:
These don't seem needed anymore.

Reply:
The x64 one is no longer needed. The arm exclusions are still needed in some form, they're matching .gitignore:250:

`[Aa][Rr][Mm]/`

I'll move them into the main file and follow the x64 pattern.